content/components/recipes/details/DetailsHeader.js

```diff
@@ -147,6 +149,24 @@ export default function DetailsHeader() {
     );
   }
 
+  let telemetryLink = null;
+  if (environment.experimenterUrl && data.experimenter_slug) {
+    const {
+      normandySlug,
+      startDate,
+      endDate,
+      status,
+    } = useExperimenterDetailsData();
+    telemetryLink = (
+      <TelemetryLink
+        endDate={endDate}
+        normandySlug={normandySlug}
+        startDate={startDate}
+        status={status}
+      />
+    );
```

Comment:
If it's ok to pass extra things into the component (and with Typescript that works better than without), you could do this much more concisely:

```suggestion
    telemetryLink = <TelemetryLink {...useExperimenterDetailsData()} />;
```